### PR TITLE
Add section documenting the meter model

### PR DESCRIPTION
**Pull request by gtfierro**

### CODE REVIEW

File: modeling/meters.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+```ttl
+:building_meter a brick:Building_Meter ;
+    brick:hasSubMeter :floor-meter-1, :floor-meter-2 .
+:floor-meter-1 a brick:Meter .
+:floor-meter-2 a brick:Meter .
+```
+
+```{warning}
+Brick does not add any properties to the submeter relationship --- it only indicates that there is one. If this is a problem, [let us know!](https://github.com/BrickSchema/Brick/issues)
```
Comment (jbkoh) [inline modeling/meters.md:38]: Both meters and submeters may have `brick:meters` relationships as described below. I think saying that exists here could avoid confusion in the future.

File: modeling/meters.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+site:array1 a brick:PV_Array .
+site:panel11 a brick:PV_Panel ;
+    brick:isPartOf site:array1 ;
+    brick:panelArea [ brick:hasUnit unit:M2 ;
+            brick:value "5"^^xsd:double ] .
+site:panel12 a brick:PV_Panel ;
+    brick:isPartOf site:array1 .
+:pv_meter   a   brick:Electrical_Meter .
+```
+
```
Comment (jbkoh) [inline modeling/meters.md:106]: An example query that works for all of those patterns would be helpful.

**Review by jbkoh — DISMISSED** [review]
It looks good overall! Two changes requested as described in each comment.

File: modeling/meters.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+## Meter Data
+
+Meters can host several points which correspond to the data produced by the meter. These point instances are related to the meter via the `brick:isPointOf` relationship. See [[metadata/entity-properties]] for some examples of how those points can be described.
```
Comment (jbkoh) [inline modeling/meters.md:109]: Please indicate `brick:hasPoint` is also possible to be kinder to newcomers.

File: modeling/meters.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+Meters
+======
+
+```{note}
+This feature is new in Brick v1.3
+```
+
+Brick provides a basic but flexible model for describing meters, submeter hierarchies, and their relationships to the building.
+Meters are equipment that measure the consumption or production of energy, steam, gas or other "substances" in the building.
```
Comment (jbkoh) [inline modeling/meters.md:21]: We can be more up front about how to use Meters as equipment by providing example points (Electricity_Usage_Sensor, Power_Sensor, etc.) possibly associated with meters. People get easily confused why meters are equipment.

File: modeling/meters.md (hunk 1 above)
Comment (gtfierro) [reply modeling/meters.md:38]: "submeter" is a relationship, not an entity type. I tried to clarify this in the new text

File: modeling/meters.md (hunk 2 above)
Comment (gtfierro) [reply modeling/meters.md:106]: Good idea! Added one

File: modeling/meters.md (hunk 3 above)
Comment (gtfierro) [reply modeling/meters.md:109]: Fixed -- thanks!

File: modeling/meters.md (hunk 4 above)
Comment (gtfierro) [reply modeling/meters.md]: Added some more text to clarify

**Comment by gtfierro** [comment]
@epaulson I moved the "data" section to the top of the doc

**Pull request merged**